bin/foreman-proxy-certs-generate

```diff
@@ -1,10 +1,10 @@
-#!/usr/bin/env ruby
+#!/usr/bin/ruby
 require 'rubygems'
 require 'kafo'
 require 'tempfile'
 
 class ForemanProxyCertsGenerate
-  INSTALLER_DIR = '.'
+  INSTALLER_DIR = "/usr/share/katello-installer-base"
```

Comment:
This can probably be left as is, the RPM packaging changes it to the production directories: https://github.com/Katello/katello-packaging/blob/master/katello-installer/katello-installer-base.spec#L100